### PR TITLE
just some labels edits

### PR DESCRIPTION
**Pull request by dariala**

### CODE REVIEW

File: js/Stack.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class Stack {
+    
     arr = [];
+
```
Comment (jozeflacko) [inline js/Stack.js:4]: question: Why you have added these empty lines? :)

File: js/Graph/BinaryTree/DepthFirstTraversal/InOrderTraversal.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
-function inOrderTraversalUsingRecursion(node, cb) {
+function inOrderTraversalUsingRecursion(node, callback) {
     if(node == null) {
         return;
     }
-    inOrderTraversalUsingRecursion(node.left, cb);
-    cb ? cb(node) : console.log(node.value);
-    inOrderTraversalUsingRecursion(node.right, cb);
+    inOrderTraversalUsingRecursion(node.left, callback);
+    callback ? callback(node) : console.log(node.value);
```
Comment (jozeflacko) [inline js/Graph/BinaryTree/DepthFirstTraversal/InOrderTraversal.js:10]: praise: it is more readable :)

File: js/test/deleteWholeTree.test.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-const {deleteWholeTree} = require('../Graph/BinaryTree/deleteWholeTree');
 const {createBinarySearchTree} = require('../Graph/BinaryTree/createBinarySearchTree');
+const {deleteWholeTree} = require('../Graph/BinaryTree/deleteWholeTree');
 
-it('deleteWholeTree', () => {
+it('delete whole tree', () => {
```
Comment (jozeflacko) [inline js/test/deleteWholeTree.test.js:4]: praise: thank you!

**Review by jozeflacko — COMMENTED** [review]
Thanks for suggestions. There is one open question

File: js/Graph/BinaryTree/DepthFirstTraversal/InOrderTraversal.js (hunk 2 above)
Comment (dariala) [reply js/Graph/BinaryTree/DepthFirstTraversal/InOrderTraversal.js:10]: you see :)

File: js/test/deleteWholeTree.test.js (hunk 3 above)
Comment (dariala) [reply js/test/deleteWholeTree.test.js:4]: gerne :=)

File: js/Stack.js (hunk 1 above)
Comment (dariala) [reply js/Stack.js:4]: my default editor did it

**Comment by dariala** [comment]
do you like my changes?

**Review by jozeflacko — APPROVED** [review]
ok

**Pull request merged**